### ISSUE TITLE
cmd/go: timing out on plan9/386 builder

### ISSUE THREAD

**Issue by 0intro** [golang/go#29822]
Since the addition of many new tests in cmd/go during the last
few weeks, the cmd/go tests are timing out on the plan9/386
builder running on GCE.

```
Test "go_test:cmd/go" ran over 20m0s limit (20m0.001784079s)
```

This is related to CL [158537](https://golang.org/cl/158537).

**Comment by 0intro** [comment]
File I/O are slow on Plan 9. There was a deadlock in `ramfs` related to the use of the lockfile package, but I fixed in November (issue #29033). Now, the cmd/go tests complete, but it takes more than 20m (the coordinator timeout) to complete.

I plan to enable SMP on the Plan 9 image, so the cmd/go tests could complete significantly faster.

**Comment by bradfitz** [comment]
Deployed & cleared the plan 9 column again. Let's see what happens.
